### PR TITLE
Add Community Ops IAM group and role

### PR DESCRIPTION
**Pull request by flamingspaz**
- Add group with RO permissions
- Add role with RW permissions for MFA users

### CODE REVIEW

**Comment by flamingspaz** [comment]
While we don't use `user-data` yet, we plan to in the future so I've explicitly denied `ec2:DescribeInstanceAttribute`.

Good catch with creating MFA devices. I've moved `iam:CreateVirtualMFADevice` and `iam:EnableMFADevice` to a policy attached to the group.

(Also, it's now using the correct Terraform resource type instead of the one for managed policies)

File: iam.tf [hunk 1]
```diff
@@ -34,3 +34,216 @@ resource "aws_iam_role_policy_attachment" "admin-access-policy" {
...
+            "route53:DeleteTrafficPolicyInstance",
+            "route53:DisassociateVPCFromHostedZone",
+            "route53:TestDNSAnswer",
+            "route53:UpdateHealthCheck",
+            "route53:UpdateHostedZoneComment",
+            "route53:UpdateTrafficPolicyComment",
+            "route53:UpdateTrafficPolicyInstance",
+            "cloudfront:CreateInvalidation",
+            "cloudfront:ListInvalidations"
+
```
Comment (johngian) [inline iam.tf]: Please remove trailing line

File: iam.tf [hunk 2]
```diff
@@ -34,3 +34,216 @@ resource "aws_iam_role_policy_attachment" "admin-access-policy" {
...
+data "aws_iam_policy_document" "community-ops-MFA-policy-document" {
+    statement {
+        effect  = "Allow"
+        actions = [
+            "iam:CreateVirtualMFADevice",
+            "iam:EnableMFADevice",
+        ]
+
+        resources = [
+            "arn:aws:iam::${var.aws_account_id}:user/$${aws:username}",
```
Comment (johngian) [inline iam.tf]: Do we need both `$$` here?

Comment (flamingspaz) [inline iam.tf]: Yep, `${aws:username}` is an AWS thing so we're escaping it.

Comment (gene1wood) [inline iam.tf]: http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_variables.html

**Pull request merged**